phpBB/phpbb/textformatter/s9e/parser.php

```diff
@@ -56,24 +50,20 @@ public function __construct(\phpbb\cache\driver\driver_interface $cache, $key, \
 
 		$this->dispatcher = $dispatcher;
 		$this->parser = $parser;
-		$this->user = $user;
 		$parser = $this;
 
 		/**
 		* Configure the parser service
 		*
 		* Can be used to:
-		*  - toggle features according to the user's preferences,
-		*  - toggle BBCodes according to the user's permissions,
```

Comment:
@s9e AFAIK extensions are using the same user object anyway, so this is not a problem.